aim2dat/strct/ext_manipulation/add_structure.py

```diff
@@ -358,6 +355,36 @@ def _add_mol(
     return new_structure, score
 
 
+def _merge_structures(host_strct, guest_strct, wrap):
+    new_structure = host_strct.to_dict()
+    new_structure["elements"] = list(new_structure["elements"])
+    if new_structure["kinds"] is not None:
+        new_structure["kinds"] = list(new_structure["kinds"])
+    else:
+        new_structure["kinds"] = [None] * len(new_structure["elements"])
+    new_structure["positions"] = list(new_structure["positions"])
+    new_structure["site_attributes"] = {
+        key: list(val) for key, val in new_structure["site_attributes"].items()
+    }
+    if len(guest_strct["site_attributes"]) > 0:
+        for site_attr in guest_strct["site_attributes"].keys():
+            if site_attr not in new_structure["site_attributes"]:
+                new_structure["site_attributes"][site_attr] = [None] * len(
+                    new_structure["elements"]
+                )
+    for idx, (el, kind, pos) in enumerate(
+        zip(guest_strct["elements"], guest_strct["kinds"], guest_strct["positions"])
+    ):
```

Comment:
idx is not used, I would suggest:

    for (el, kind, pos) in (
        zip(guest_strct["elements"], guest_strct["kinds"], guest_strct["positions"])
    ):

Reply:
Thanks for the catch